scss/_patterns_navigation.scss

```diff
@@ -274,6 +297,29 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     list-style: none;
     margin: -1px 0 0 0;
     padding: 0;
+    position: relative;
+
+    // add separator between navigation items
+    &::before {
```

Comment:
Would it be possible to extract this into some `%placeholder` to reuse it?

Something like:

```scss
%vf-navigation-separator {
  &::before {
   …
   }
 }
 ```
 
 And then just `@extend %vf-navigation-separator` where needed?

Reply:
@bartaz done!